core/src/main/java/org/openstack4j/openstack/telemetry/domain/CeilometerStatistics.java

```diff
@@ -34,17 +46,22 @@
 
 	private Integer period;
 
-	@JsonProperty("period_start")
+   @JsonProperty("period_start")
```

Comment:
Would
```
@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
@JsonProperty("period_start")
private Date durationStart;
```
instead of adding a new property work?
Then you could also revert the changes made to the `getDurationStart()` and `getDurationEnd()`.

Please add a test for this.